mne/data/coil_def.dat

```diff
@@ -3,7 +3,7 @@
 #
 #       Copyright 2005 - 2019
 #
-#       Matti Hämäläinen
+#       Matti Hamalainen
```

Comment:
why this?

Reply:
Matti generates this with his C code, I'm guessing it doesn't get along well with unicode